src/Copy/BrandCopy.elm

```diff
@@ -566,12 +584,18 @@ brandCopy key =
         StoryThree3ImageAlt ->
             CopyText "✨"
 
+        StoryThree3Message ->
```

Comment:
Too much of a pain now but I'd name all these vars in the format `Story3Message3`

Reply:
The reason we didn't is the ones that end in H3 etc. (I think) - but that would make sense... Maybe we can sneak it in to a refactor before Seap/ Samaritans?